convert_old_alw_to_new_structure.py

```diff
@@ -0,0 +1,26 @@
+import csv
+#csv file from which the original data is read
+with open('hr.contract (8).csv') as csvfile1: 
+	#csv file to which new data is written
+	with open('csvfile.csv','w') as csvfile2:
+		#the keys in the new csv file
+		fieldnames = ['External ID', 'Allowances', 'Allowances/Amount']
+		#the row values in the old csv file are assigned to "reader"
+		reader = csv.DictReader(csvfile1)  
+		#write the headers in the new csv file
+		writer = csv.DictWriter(csvfile2,fieldnames=fieldnames)
+		writer.writeheader()
+		#main data conversion 
+		for row in reader: 
+			writer.writerow({'External ID': row['External ID'],  
+				'Allowances': '__export__.hr_payroll_allowance_line_1', 
+				'Allowances/Amount': row['Medical']}) 
+			writer.writerow({'External ID': '',  
+				'Allowances': '__export__.hr_payroll_allowance_line_2', 
+				'Allowances/Amount': row['Transport']}) 
+			writer.writerow({'External ID': '',  
+				'Allowances': '__export__.hr_payroll_allowance_line_3', 
+				'Allowances/Amount': row['Rent']}) 
+
```

Comment:
Please do not hard code fieldnames, this can easily be read for the the csv.DictReader instance e.g.
```python
reader = csv.DictReader(x) 
fieldnames = reader.fieldnames
```
As explained yesterday, create a dict that stores the mapping of Fieldnames in sourcefiel to external IDs in destination file and use that to build the destination file.

Also if I run this it won't give the output file in the format I sent you... it is important that the format is exactly the same